doc/API.md

```diff
@@ -989,11 +990,34 @@ lists the status of all files.
 }
 ```
 
+#### /renter/file/*__siapath__ [GET]
+
+lists the status of specified file.
+
+###### JSON Response [(with comments)](/doc/api/Renter.md#json-response-4)
+```javascript
+{
+  "file": [
+    {
```

Comment:
I don't think that this is the correct format since the `RenterFile` struct doesn't contain a slice of files but instead only a single file. You should double check that using `curl` but I think `[` and `]` are not necessary.